### PR TITLE
Change to be exposed plugin method. Closes #3.

### LINKED ISSUE #3: Rework
Expose the method for collection all of the hapi data as a plugin method and let the consumer use/display it as they so choose.

Related hapi issue: https://github.com/hapijs/hapi/issues/2777

### PR DESCRIPTION
**Pull request by arb**
- Removed all routes the plugin adds
- Clone `process.env`
- Updated README with changes

Maybe not complete, but wanted to get the conversation moving.

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -12,13 +8,13 @@ function constructResponse (server) {
     argv: Hoek.clone(process.argv),
     execArgv: Hoek.clone(process.execArgv),
     cwd: process.cwd(),
-    // This should probably be a clone(), but Hoek is currently breaking
-    // when working with process.env on 0.10
-    env: process.env
+    env: Hoek.clone(process.env)
```
Comment (cjihrig) [inline lib/index.js:11]: Has the new version of Hoek been published with the fix?

File: lib/index.js [hunk 2]
```diff
@@ -12,13 +8,13 @@ function constructResponse (server) {
...
   };
   var connections = server.connections.map(function (connection) {
     var plugins = {};
 
+    // Don't use _registrations.
```
Comment (cjihrig) [inline lib/index.js]: Can you make this a `TODO` comment?

File: test/fixtures/server.js [hunk 3]
```diff
@@ -1,6 +1,8 @@
 'use strict';
 var Hapi = require('hapi');
 var HapiSetup = require('../../lib');
+var PluginFoo = require('./plugins').Foo;
```
Comment (cjihrig) [inline test/fixtures/server.js]: Can you do the `require()` once and store it in `Plugins`

File: lib/index.js (hunk 1 above)
Comment (arb) [inline lib/index.js:11]: Yes. Travis tests pass so, we're good.

https://github.com/hapijs/hoek/issues/162

File: README.md [hunk 4]
```diff
@@ -5,24 +5,34 @@
...
 ![devDependencies](http://img.shields.io/david/dev/continuationlabs/hapi-setup.svg)
 
-hapi plugin for viewing the server configuration. Provides information such as the version of Node running, the hapi server connections, routing tables per connection, and plugin information.
+hapi plugin that exposes a `setup` method for retrieving the server configuration. Provides information such as the version of Node running, the hapi server connections, routing tables per connection, and plugin information.
 
-## Configuration Options
+## Example Usage
 
-`hapi-setup` supports the following options via the plugin's `register()` function.
+The example below registers the `hapi-setup` plugin. In the handler for the "/about" route, the `hapi-setup.setup()` function is executed, which returns an object, and the result is then used to reply to the incomming request.
```
Comment (cjihrig) [inline README.md]: Can you put `/about` in backticks instead of quotes

Comment (cjihrig) [inline README.md]: `incomming` is spelt wrong

File: README.md [hunk 5]
```diff
@@ -5,24 +5,34 @@
...
+```js
```
Comment (cjihrig) [inline README.md:14]: Please add a blank line before this line.

**Pull request merged**